src/Hosting.Services/HostBuilderExtensions.cs

```diff
@@ -101,16 +110,20 @@ public static IServiceCollection AddOmexServiceFabricDependencies<TContext>(this
 						options.ValidateOnBuild = true;
 						options.ValidateScopes = true;
 					})
-					.ConfigureLogging(builder => builder.AddOmexLogging())
 					.Build();
```

Comment:
Isnt it redundant? 
AddOmexServiceFabricDependencies() will call AddOmexServices() will call AddOmexLogging()

Reply:
It may be redundant, but it's not obvious. There may also be functional differences, e.g. you are removing a call to ILoggingBuilder.AddConfiguration. Besides, this change is out of scope for obsoletion task.

Reply:
Reverted